### ISSUE TITLE
Decomposition of actors as sub-actors

### ISSUE THREAD

**Issue by cp-api** [cp-api/bugs-archive#192]
Currently capella does not break down actors as "sub-actors", which in some cases forced to skew the model. For example, through modeling of launcher avionics, we need to model the interfaces between electrical system (which is a part of avionic) and the upper stage engine (which is part of the sub-propulsion system of the launcher).

If we apply the ARCADIA methodology implemented in capella, we can only have one level of decomposition for the engine, for all the allocated functions of this engine.

But in our model we would be able to handle interfaces with it in more details, because we know this actor (the engine) since it is already developed.
Today the recommended solution is to integrate some of the actor in our system.
In conclusion, the need is to decompose actors as "sub-actors" in order to improve the level of description for the interfaces between the system and the actors.

`ECLIPSE-554953` `POLARSYS-409` `@sch` `2015-07-23`
